build/pmm-client-docker/entrypoint.sh

```diff
@@ -35,54 +35,143 @@ wait_for_port() {
     exit 1
 }
 
-if [ -z "${PMM_SERVER}" ]; then
-    echo PMM_SERVER is not specified. exiting
-    exit 1
-fi
-if [ -n "${PMM_USER}" ]; then
-    ARGS+=" --server-user ${PMM_USER}"
-fi
-if [ -n "${PMM_PASSWORD}" ]; then
-    ARGS+=" --server-password ${PMM_PASSWORD}"
-fi
-
-PMM_SERVER_IP=$(ping -c 1 "${PMM_SERVER/:*/}" | grep PING | sed -e 's/).*//; s/.*(//')
-SRC_ADDR=$(ip route get "${PMM_SERVER_IP}" | grep 'src ' | sed -e 's/.* src //; s/ .*//')
-CLIENT_NAME=${DB_HOST:-$HOSTNAME}
-
-wait_for_url "https://${PMM_USER}:${PMM_PASSWORD}@${PMM_SERVER}/v1/status/leader" "127.0.0.1:8300"
-
-pmm-admin config \
-    --skip-root \
+pmm2_start() {
+    AGENT_CONFIG_FILE=/usr/local/percona/pmm2/config/pmm-agent.yaml
+    rm -f ${AGENT_CONFIG_FILE}
+    if [ -n "${PMM_USER}" ]; then
+        ARGS+=" --server-username=${PMM_USER}"
+    fi
+
+    pmm-agent setup \
     --force \
```

Comment:
style fix - tabulation (4 spaces added)
```suggestion
        --force \
```

Reply:
done